src/components/InlineLoading/InlineLoading.js

```diff
@@ -30,6 +30,11 @@ export default class InlineLoading extends React.Component {
      */
     description: PropTypes.string,
 
+    /**
+     * Specify the description for the inline loading text
+     */
+    iconDescription: PropTypes.string,
```

Comment:
What's the difference between this and description? Can we just use description instead of adding a new prop? Looks like you might have accidentally not updated the comment block

Reply:
`description` is meant to be a node instead of a string, and `iconDescription` is for the SVG title in `<Loading />`